frontend/src/scenes/web-analytics/PageReports.tsx

```diff
@@ -0,0 +1,93 @@
+
+import { pageReportsLogic } from './pageReportsLogic'
+import { SectionsList } from './WebAnalyticsDashboard'
+
+export function PageReportsFilters(): JSX.Element {
+    const { pagesUrls, pageUrl, isLoading, stripQueryParams, dateFilter } = useValues(pageReportsLogic)
+    const { setPageUrl, setPageUrlSearchTerm, toggleStripQueryParams, loadPages, setDates } =
+        useActions(pageReportsLogic)
+
+    const options = pagesUrls.map((option: { url: string; count: number }) => ({
+        key: option.url,
+        label: option.url,
+        labelComponent: (
+            <div className="flex justify-between items-center w-full">
+                <span className="truncate">{option.url}</span>
+                <span className="text-muted ml-2">{option.count.toLocaleString()}</span>
+            </div>
+        ),
+    }))
+
+    return (
+        <div className="flex flex-col gap-2">
+            <div className="flex items-center gap-2">
+                <div className="flex-1">
+                    <LemonInputSelect
+                        allowCustomValues={false}
+                        placeholder="Click or type to see top pages"
+                        loading={isLoading}
+                        size="small"
+                        mode="single"
+                        value={pageUrl ? [pageUrl] : null}
+                        onChange={(val: string[]) => setPageUrl(val.length > 0 ? val[0] : null)}
+                        options={options}
+                        onInputChange={(val: string) => setPageUrlSearchTerm(val)}
+                        data-attr="page-url-search"
+                        onFocus={() => loadPages('')}
+                        className="max-w-full"
+                    />
+                </div>
+                <Tooltip title="Strip query parameters from URLs (e.g. '?utm_source=...'). This will match the base URL regardless of query parameters.">
+                    <div className="inline-block">
+                        <LemonSwitch
+                            checked={stripQueryParams}
+                            onChange={toggleStripQueryParams}
+                            label="Strip query params"
+                            size="small"
+                            bordered
+                        />
+                    </div>
+                </Tooltip>
+                <div>
+                    <DateFilter
+                        dateFrom={dateFilter.dateFrom}
+                        dateTo={dateFilter.dateTo}
+                        onChange={(fromDate, toDate) => setDates(fromDate, toDate)}
+                    />
+                </div>
+            </div>
+        </div>
+    )
+}
+
+export function PageReports(): JSX.Element {
+    const { hasPageUrl, sections } = useValues(pageReportsLogic)
+
+    if (!hasPageUrl) {
+        return (
+            <div className="space-y-2 mt-2">
+                <ProductIntroduction
+                    productName="PAGE REPORTS"
```

Comment:
I don't see this as uppercase anywhere else, is this correct?
![image](https://github.com/user-attachments/assets/72d35044-4f08-48a5-a8be-45241cd61a5b)